scrapers/LustCinema.yml

```diff
@@ -0,0 +1,41 @@
+name: LustCinema
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lustcinema.com
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    common: &com
+      $cast: //div[contains(concat(' ',normalize-space(@class),' '),'relative px-1')]
+      $script: //script[contains(.,"window.__NUXT")]/text()
+    scene:
+      Title: &title //h1
+      Date: &date
+        selector: //script[@type='application/ld+json' and contains(., 'dateCreated')]/text()
+        postProcess:
+          - replace:
+              - regex: .*dateCreated(?:[\"\s:])+(\d+-\d+-\d+) .*
+                with: $1
+          - parseDate: 2006-01-02
+      Details: &details
+        selector: $script
+        postProcess:
+          - replace:
+              - regex: .*,synopsis_clean(?:[\s:])+\"([^\"]*)\",.*
+                with: $1
+      Performers:
+        Name:
+          selector: $cast//a[contains(.,'Performer')]/div[contains(concat(' ',normalize-space(@class),' '),'mt-1')]
```

Comment:
is there a need for the normalize space,concat ?
```yaml
      Performers:
        Name: //div[@id="contributorImage"]//a[contains(@href,"/cast/")]/@title
```
seems to work and is simpler

Reply:
Had issues making contains() work with class, will take a look at your solution